scripts/lib/wic/canned-wks/mkefidisk-multiboot.wks

```diff
@@ -2,12 +2,12 @@
 # long-description: Creates a partitioned EFI disk image that the user
 # can directly dd to boot media.
 
-part /boot --source bootimg-efi --sourceparams="loader=grub-efi" --ondisk sda --label msdos --active --align 1024
+part /boot --source bootimg-efi --sourceparams="loader=grub-efi" --ondisk sda --label msdos --active --align 1024 --use-uuid
 
-part / --source rootfs --ondisk sda --fstype=ext4 --label platform --align 1024
+part / --source rootfs --ondisk sda --fstype=ext4 --label platformA --align 1024 --use-uuid
```

Comment:
@TSAnsari ?

Reply:
As stated in the commit message. More work is needed to make this work with swupdate since grub still refers to /dev/sda

Reply:
Yes, I've read the commit message, but it'll break swupdate, which we don't want.

Reply:
I think what @sashko is trying to say (?) is that @TSAnsari has been working on fixing this (hardcoded device names) recently. Check with @TSAnsari . :)